lib/fetch/rss/fetchRssFeed.ts

```diff
@@ -11,7 +11,11 @@ type CustomItem = {
 
 const parser: Parser<CustomFeed, CustomItem> = new Parser({
   customFields: {
-    feed: ['podcast:value', 'itunes:type'],
+    feed: [
+      // @ts-ignore
+      ['podcast:value', 'podcast:value', { keepArray: true }],
```

Comment:
`keepArray`! that's a good option, didn't know about that one...

Reply:
Had to dig deep in the docs to find it.